app/src/js/components/Executions/index.js

```diff
@@ -21,10 +21,27 @@ class Executions extends React.Component {
     this.displayName = strings.executions;
   }
 
+  renderHeader () {
+    const { pathname } = this.props.location;
+    const showDatePicker = pathname === '/executions';
+
+    if (showDatePicker) {
+      return <DatePickerHeader onChange={this.query} heading={strings.executions}/>;
+    } else {
+      return (
+        <div className='content__header'>
+          <div className='row'>
+            <h1 className='heading--xlarge'>{strings.executions}</h1>
+          </div>
+        </div>
+      );
+    }
+  }
+
   render () {
     return (
       <div className='page__workflows'>
-        <DatePickerHeader onChange={this.query} heading={strings.executions}/>
+        {this.renderHeader()}
```

Comment:
Just curious why we're conditionally displaying `DatePickerHeader` here? We don't do this for an individual collection page, so not sure why we're doing it here.

Reply:
So in discussions about this when I started the ticket, the DatePicker doesn't actually affect anything on the individual execution status page, and it was suggested that it didn't need to be on the page. So I removed it.

Reply:
Gotcha, that's fair enough